docs/faq.md

```diff
@@ -18,30 +18,124 @@
 -->
 
 # FAQs
-* **Auto Compaction not Working**
 
-    The Property carbon.enable.auto.load.merge in carbon.properties need to be set to true.
+* [Can we preserve Segments from Compaction?](#can-we-preserve-segments-from-compaction)
+* [Can we disable horizontal compaction?](#can-we-disable-horizontal-compaction)
+* [What is horizontal compaction?](#what-is-horizontal-compaction)
+* [How to enable Compaction while data loading?](#how-to-enable-compaction-while-data-loading)
+* [Where are Bad Records Stored in CarbonData?](#where-are-bad-records-stored-in-carbondata)
+* [What are Bad Records?](#what-are-bad-records)
+* [Can we use CarbonData on Standalone Spark Cluster?](#can-we-use-carbondata-on-standalone-spark-cluster)
+* [What versions of Apache Spark are Compatible with CarbonData?](#what-versions-of-apache-spark-are-compatible-with-carbondata)
+* [Can we Load Data from excel?](#can-we-load-data-from-excel)
+* [How to enable Single Pass Data Loading?](#how-to-enable-single-pass-data-loading)
+* [What is Single Pass Data Loading?](#what-is-single-pass-data-loading)
+* [How to specify the data loading format for CarbonData ?](#how-to-specify-the-data-loading-format-for-carbondata)
+* [How to resolve store location can’t be found?](#how-to-resolve-store-location-can-not-be-found)
+* [What is carbon.lock.type?]()
+* [How to enable Auto Compaction?](#how-to-enable-auto-compaction)
+* [How to resolve Abstract Method Error?](#how-to-resolve-abstract-method-error)
+* [Getting Exception on Creating a View](#getting-exception-on-creating-a-view)
+* [Is CarbonData supported for Windows?](#is-carbondata-supported-for-windows)
+* [How to handle Bad Records?](#how-to-handle-bad-records)
 
-* **Getting Abstract method error**
+## Can we preserve Segments from Compaction?
+If you want to preserve number of segments from being compacted then you can set the property  **carbon.numberof.preserve.segments**  equal to the **value of number of segments to be preserved**.
 
-    You need to specify the spark version while using Maven to build project.
+Note : *No segments are preserved by Default.*
 
-* **Getting NotImplementedException for subquery using IN and EXISTS**
+## Can we disable horizontal compaction?
+Yes, to disable horizontal compaction, set **carbon.horizontal.compaction.enable** to ``FALSE`` in carbon.properties file.
```

Comment:
I don't suggest adding the FAQ about disable horizontal compaction, users don't need to set the parameter and this can be controlled automatically by system.